Author: Kai-Chung Yan <seamlikok@gmail.com>
Last-Update: 2016-06-07
Description: Prepare the build script
 This patch modifies build.gradle so that it can be used by Debian's build
 system. It does the following:
   * Set artifactId, groupId and version
   * Specify offline dependencies
   * Generate POMs
   * Set Java version to 1.6
--- a/external/jsilver/build.gradle
+++ b/external/jsilver/build.gradle
@@ -1,5 +1,11 @@
 apply plugin: 'java'
 
+archivesBaseName = 'jsilver'
+group = 'com.google.jsilver'
+version = "android-$androidVersion"
+sourceCompatibility = 1.6
+targetCompatibility = 1.6
+
 sourceSets {
     main {
         java {
@@ -14,5 +20,7 @@
 }
 
 dependencies {
-    compile project(':guava')
+    compile 'com.google.guava:guava:debian'
 }
+
+apply from: 'debian/generatePom.gradle'
\ No newline at end of file
